modules/order/src/Form/OrderReassignForm.php

```diff
@@ -22,20 +24,44 @@ class OrderReassignForm extends FormBase {
   protected $order;
 
   /**
+   * The order assignment service.
+   *
+   * @var \Drupal\commerce_order\OrderAssignment
+   */
+  protected $OrderAssignment;
+
+  /**
+   * The user storage.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $userStorage;
+
+  /**
    * Constructs a new OrderReassignForm object.
    *
    * @param \Drupal\Core\Routing\CurrentRouteMatch $current_route_match
    *   The current route match.
+   * @param \Drupal\commerce_order\OrderAssignment $order_assignment
+   *   The order assignment service.
+   * @param \Drupal\Core\Entity\EntityStorageInterface $user_storage
+   *   The user storage.
    */
-  public function __construct(CurrentRouteMatch $current_route_match) {
+  public function __construct(CurrentRouteMatch $current_route_match, OrderAssignment $order_assignment, EntityStorageInterface $user_storage) {
     $this->order = $current_route_match->getParameter('commerce_order');
+    $this->OrderAssignment = $order_assignment;
+    $this->userStorage = $user_storage;
   }
 
   /**
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container) {
-    return new static($container->get('current_route_match'));
+    return new static(
+      $container->get('current_route_match'),
+      $container->get('commerce_order.order_assignment'),
+      $container->get('entity.manager')->getStorage('user')
```

Comment:
It should be `entity_type.manager`, and usually we just inject that, then in constructor pluck out the storages we need.